src/Linux/Containers/Impl/Docker/LinuxDockerService.cs

```diff
@@ -85,46 +72,12 @@ public class LinuxDockerService : LocalDockerService, IContainerService {
         }
 
         protected override LocalDocker GetLocalDocker() {
-            _docker = _docker ?? GetLocalDocker(_services);
-            CheckIfServiceIsRunning();
+            _docker = _docker ?? _dockerFinder.GetLocalDocker();
+            _dockerFinder.CheckIfServiceIsRunning();
```

Comment:
Then we will always get service not running error. Instead of service is not installed error when docker is not installed.